src/lib/customers/20minuten-en.json

```diff
@@ -0,0 +1,54 @@
+		"topics": [
+			{
+				"title": "Infrastructure on AWS",
+				"body": "bespinian has been instrumental in provisioning the infrastructure on AWS. We played a key role in writing and applying the appropriate Terraform modules and setting up the Kubernetes clusters on which the various microservices are hosted. An important part of this was also the correct configuration of the caching layer (Amazon CloudFront) to keep the costs manageable with the large user:inside number of 20 Minuten. bespinian did a lot of the technical implementation and automation, but also the communication between the development and infrastructure teams."
+			},
+			{
+				"title": "Kubernetes and container configuration",
+				"body": "An exciting task was the configuration of the Kubernetes cluster (on Amazon EKS). The architecture as well as the implementation of it still bears bespinian's signature. It was about architectural decisions, but also about the implementation of the provisioning and configuration of the cluster via Terraform. Subsequently, it was about the automation of the deployments of the individual services via Helm and GitHub Actions. Bespinian was allowed to participate in all these sub-areas. We also played a major role in coordinating the individual teams and applications in the sense of a DevOps migration."
+			},
+			{
+				"title": "Application Development",
+				"body": "The various microservices of the backend and the frontend were co-designed by bespinian, both architecturally and in terms of software development. As experienced Go developers, we were able to contribute to the design of the application code itself, but also to its automation, deployment and communication between the services."
+			},
+			{
+				"title": "Monitoring and Alerting",
+				"body": "Monitoring the applications in terms of a DevOps transformation was very important, especially in the initial phase after the migration. A large number of users were migrated to the new applications and sometimes produced access patterns that were difficult to predict, which we first had to understand in order to provision the appropriate resources and produce bug fixes. Prometheus and Grafana played a major role in this process to get an overview of what is happening and where which requests are being made."
+			},
+			{
+				"title": "Caching",
+				"body": "An important part of bespinian's contribution was the configuration of the CloudFront caching layer. Due to the high number of requests, most of which are read-only, we were able to save very large monthly contributions by clever use of the cache. This allowed the containers to be used in relatively small numbers of replicas. The optimal configuration of the cache with so many different access patterns and clients (20 minute website, different mobile applications and CMS) was a big challenge, where bespinian was allowed to take the lead."
```

Comment:
I would replace `The optimal configuration of the cache with so many different access patterns and clients (20 minute website, different mobile applications and CMS) was a big challenge, where bespinian was allowed to take the lead.` by `The optimal configuration of the cache with so many different access patterns and clients (20 minute website, different mobile applications and CMS) was a big challenge. bespinian was responsible for tackling this challenge and providing a solution to the 20 Minuten team.`